tests/imposter/private-api.yaml

```diff
@@ -206,7 +206,7 @@ components:
       properties:
```

Comment:
This file is sourced from https://github.com/govuk-one-login/ipv-cri-kbv-hmrc-api/blob/main/infrastructure/private-api.yaml so any changes suggested here should be made there first. (There's tech debt to retrieve this as a `post-install` step).

However, you shouldn't need to change this file for Imposter, as all the scenarios should be covered by the [private-api-config.yaml](https://github.com/govuk-one-login/ipv-cri-kbv-hmrc-front/blob/main/tests/imposter/private-api-config.yaml). So I think there's something falling through the gaps, and you're hitting the OpenAPI example instead.